pkg/appconsts/global_consts.go

```diff
@@ -67,3 +67,12 @@ func UpgradeHeightDelay() int64 {
 func HashLength() int {
 	return hashLength
 }
+
+// The following consts are not consensus breaking and will be applied straight after this binary is started.
+const (
+	// SdkMsgTransactionCap is the maximum number of SDK messages, aside from PFBs, that a block can contain.
+	SdkMsgTransactionCap = 200
+
+	// PFBTransactionCap is the maximum number of PFB messages a block can contain.
+	PFBTransactionCap = 600
+)
```

Comment:
Can we include rationale for these numbers in the description. Ideally we also merge the benchmarks PR and link to those.

Reply:
I also feel like these numbers are overly conservative based on looking at the results. I had expected values in the low thousands.

Reply:
For the rationale, it will be included after merging this PR and I will add comments, results and bencharmks etc.

For the numbers, the initial tests on my computer. But then we re-run the experiments on recommended validator config: 4cpu 16gb ram and we got lower numbers. So we reduced those to reflect a 0.25sec execution on that low config validator

Reply:
resolving so we merge. @rach-id do you mind creating a PR for the benchmark code if you haven't already? we could also include the results from the last run to compare results